android/app/src/main/java/ru/spbstu/amd/learnbraille/screens/practice/PracticeFragment.kt

```diff
@@ -82,12 +105,31 @@ class PracticeFragment : Fragment() {
             viewModel.onCorrectComplete()
         })
 
+        hintButton.setOnClickListener {
+            viewModel.onHint()

```

Comment:
This method should be called directly from xml, ui actions should be handled by Observer

Reply:
I tried (see previous commit) and it led to strange behaviour: when I visited Help and then returned to the letter, application invoked this method: showed toast, filled dots

Reply:
Ok, I'll try to fix it